installation/README.md

```diff
@@ -0,0 +1,161 @@
+# Production deployment using docker
+
+### Setup Letsencrypt Nginx Proxy Companion
+
+DNS needs to be configured for following to work
+
+Read more: https://github.com/evertramos/docker-compose-letsencrypt-nginx-proxy-companion
+
+```sh
+cd $HOME
+git clone https://github.com/evertramos/docker-compose-letsencrypt-nginx-proxy-companion.git
+cd docker-compose-letsencrypt-nginx-proxy-companion
+cp .env.sample .env
+./start.sh
+```
+
+### Clone frappe_docker repository
+
+```sh
+cd $HOME
+git clone https://github.com/frappe/frappe_docker.git
+cd frappe_docker
+cp installation/env-example .env
+
+# make directory for sites
+mkdir installation/sites
+```
+
+### Setup Environment Variables
+
+Environment variables are set to pass secret and variable data.
+
+If `env-example` is copied to `.env` following values are set.
+
+- `VERSION=edge` set version tag or latest for major version e.g. v12.3.0, v12
+- `MYSQL_ROOT_PASSWORD=admin`, set mariadb root password (bootstraps a mariadb container with this root password). If managed database mariadb is used NO need to set the password here
+- `MARIADB_HOST=mariadb` set hostname to `mariadb` in case of docker container for mariadb is used. In case managed db is used set the hostname/IP/domain name here
```

Comment:
When managed DB is used mariadb container shouldn't be created.

Reply:
Thunderbottom was trying to make a wrapper script based on containerized services instead of combination of managed and container services.

I'll separate docker-compose-common to redis and mariadb and update things in README later